ansible/roles/run-test-target/tasks/test-collection-method.yml

```diff
@@ -65,6 +70,7 @@
   - name: Run integration tests
     become: "{{ runtime_as_root }}"
     shell: |
+      echo '/tmp/core_dumps/core_%t' | sudo tee /proc/sys/kernel/core_pattern
```

Comment:
Can we make this its own step?

Reply:
Or perhaps combine it with the "Make core dumps directory" step above?

Reply:
Done